src/components/RecordDetail/MediaContents.tsx

```diff
@@ -0,0 +1,37 @@
+import { useState } from 'react';
+import { useTranslations } from '@i18n/client';
+import { MediaGallery } from '@performant-software/core-data';
+
+interface Props {
+  data: any[];
+  uuid: string;
+}
+
+const MediaContents = (props: Props) => {
```

Comment:
Here we should be displaying collections of manifests, much like we do in the detail panels.